sys/netlink/netlink_snl.h

```diff
@@ -102,7 +102,7 @@ lb_allocz(struct linear_buffer *lb, int len)
 	len = roundup2(len, alignof(__max_align_t));
 	if (lb->offset + len > lb->size)
 		return (NULL);
-	void *data = (void *)(lb->base + lb->offset);
+	char *data = (lb->base + lb->offset);
 	lb->offset += len;
 	return ((char *)data);
```

Comment:
Since you've used char *data ... you can safely remove this cast

Reply:
This looks to be the only active comment, and the only thing that has caught my eye that's in the way of committing.